src/Umbraco.Core/Packaging/PackageDataInstallation.cs

```diff
@@ -692,10 +693,45 @@ private IContentType UpdateContentTypeFromXml(XElement documentType, IContentTyp
             UpdateContentTypesAllowedTemplates(contentType, infoElement.Element("AllowedTemplates"), defaultTemplateElement);
             UpdateContentTypesPropertyGroups(contentType, documentType.Element("Tabs"));
             UpdateContentTypesProperties(contentType, documentType.Element("GenericProperties"));
+            UpdateHistoryCleanupPolicy(contentType, documentType.Element("HistoryCleanupPolicy"));
 
             return contentType;
         }
 
+        private void UpdateHistoryCleanupPolicy(IContentType contentType, XElement element)
+        {
+            contentType.HistoryCleanup ??= new HistoryCleanup();
+
+            if (element == null)
+            {
+                contentType.HistoryCleanup = new HistoryCleanup();
+                return;
+            }
```

Comment:
This should not alter existing configuration if the element is missing:
```suggestion
            if (element == null)
            {
                // Keep existing configuration (if any)
                return;
            }
            
            contentType.HistoryCleanup ??= new HistoryCleanup();
```

Reply:
dcbd51cf